.github/workflows/haskell.yml

```diff
@@ -91,3 +91,13 @@ jobs:
       - run: cabal v2-test all --disable-optimization $CONFIG --test-options "--fail-on-focus"
       - run: cabal v2-bench all --disable-optimization $CONFIG
       - run: cabal v2-sdist all
+  cabal-fmt:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: haskell-actions/setup@v2
+        with:
+          ghc-version: "9.6"
+          cabal-version: "3.6"
+      - run: cabal install cabal-fmt
+      - run: cabal-fmt --check **/*.cabal
```

Comment:
It looks like we can add `cabal-fmt` to `restyled` - do you mind adding it there instead?

Reply:
I was not sure which one to add, but I made the decision to add it to the workflow that runs cabal, based on the analogy that the restyled workflow may only want to run tools made by restyled.
Of course, I will make the change according to the maintainer's decision.